k-distribution/src/main/scripts/lib/checkJava

```diff
@@ -36,7 +36,7 @@ else
     setarch
     JAVA=$NG
   else
-    version=$("$_java" -version 2>&1)
+    version=$("$_java" --version)
```

Comment:
For some reason, `java --version` prints to `stdout` and `-version` to `stderr`. The two-dash version appears to be the more consistent format as well.